include/wsrep/view.hpp

```diff
@@ -111,6 +111,12 @@ namespace wsrep
             return (members_.empty() && own_index_ == -1);
         }
 
+        void fix_own_index(ssize_t i)
```

Comment:
There should not be need to fix own index in view. If the provider passes a view with invalid own index, the provider should be fixed instead.